MAINTAINERS

```diff
@@ -1,3 +1,5 @@
 Joel Speed <joel.speed@hotmail.co.uk> (@JoelSpeed)
-Henry Jenkins <henry@henryjenkins.name> (@steakunderscore)
 Nick Meves <nick.meves@greenhouse.io> (@NickMeves)
```

Comment:
Can we update Nick's email since he no longer has access to this?
@braunsonm Can we add you here?

Reply:
Thanks Jan reached out but I don't have an email I would put here. We can skip me for now!

Reply:
updated Nick's email